backstop/tests/wizard.html

```diff
@@ -29,192 +29,201 @@
     <theme-button></theme-button>
     <h1>Wizard</h1>
     <hr />
-    <h2 id="default">Default</h2>
 
+    <h2>Default</h2>
     <section id="demo-default">
-      <div class="iui-wizards">
-        <div class="iui-wizards-wrapper">
-          <span class="iui-wizards-step iui-wizards-step-completed iui-clickable">
-            <span class="iui-wizards-step-title">First step</span>
-            <span class="iui-wizards-step-track iui-wizards-step-track-after"></span>
-            <span
-              class="iui-wizards-step-circle"
-              tabindex="0"
-            >1</span>
-          </span>
 
-          <span class="iui-wizards-step-track iui-wizards-step-track-main iui-wizards-step-track-filled"></span>
-
-          <span class="iui-wizards-step iui-wizards-step-completed iui-clickable">
-            <span class="iui-wizards-step-track iui-wizards-step-track-before"></span>
-            <span class="iui-wizards-step-title">Completed step</span>
-            <span class="iui-wizards-step-track iui-wizards-step-track-after"></span>
-            <span
-              class="iui-wizards-step-circle"
-              tabindex="0"
-            >
-              <span>2</span>
-            </span>
+      <div class="iui-wizard">
+        <nav
+          class="iui-wizard-container"
+          aria-label="Progress"
```

Comment:
When everything else gets finalized, we should consider adding some aria attributes.

I would expect `role=tablist` on the wizard, `role=tab` on the steps, `aria-selected` on the current step, and maybe `role=tabpanel` on the content.